integration/test/Test/FeatureFlags/User.hs

```diff
@@ -0,0 +1,70 @@
+module Test.FeatureFlags.User where
+
+import qualified API.BrigInternal as I
+import API.Galley
+import qualified API.GalleyInternal as I
+import SetupHelpers
+import Test.FeatureFlags.Util
+import Testlib.Prelude
+
+testFeatureConferenceCallingForUser :: App ()
+testFeatureConferenceCallingForUser = do
```

Comment:
are there tests for lock_status for conf calling?

Reply:
There are now.